indico/modules/users/controllers.py

```diff
@@ -213,13 +213,15 @@ def _process(self):
         if self.user.affiliation_link:
             current_affiliation = AffiliationSchema().dump(self.user.affiliation_link)
         has_predefined_affiliations = Affiliation.query.filter(~Affiliation.is_deleted).has_rows()
-        extra_data = values_from_signal(signals.users.personal_data.send(self, user=self.user), as_list=True)
+        extra_data = {}
+        for data in values_from_signal(signals.users.extra_personal_data.send(self.user), as_list=True):
+            extra_data.update(data)
         return WPUserPersonalData.render_template('personal_data.html', 'personal_data', user=self.user,
                                                   titles=titles, user_values=user_values, locked_fields=locked_fields,
                                                   locked_field_message=multipass.locked_field_message,
                                                   current_affiliation=current_affiliation,
                                                   has_predefined_affiliations=has_predefined_affiliations,
-                                                  extra_data=extra_data[0] if extra_data else None)
+                                                  extra_data=extra_data if extra_data else None)
```

Comment:
```suggestion
                                                  extra_data=extra_data)
```

it's always a dict now

Reply:
or do you want it to be None if it's empty? `extra_data=(extra_data or None)`

but maybe an empty dict is nicer?

Reply:
Done.